examples/amp-access-fewcents.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html ⚡ lang="en">
+  <!-- prettier-ignore -->
+  <head>
+    <meta charset="utf-8">
+    <title>amp-access-fewcents</title>
+    <script id="amp-access" type="application/json">
+      {
+        "noPingback":true,
+        "vendor": "fewcents",
+        "fewcents": {
+          "contentSelector": "amp-access-fewcents-dialog"
+        }
+      }
+    </script>
+    <link rel="canonical" href="/examples/amp-access-fewcents.html">
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <style amp-custom>
+      amp-access-fewcents {
```

Comment:
This isn't doing anything?

Reply:
This has been auto generated while creating the extension template, I will check on this and update it.

Reply:
Changes done.